src/components/landing/HeadArticle/HeadArticle.js

```diff
@@ -21,40 +29,41 @@ const imagesPreload = [
   image8,
 ];
 
-class HeadArticle extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      currentImage: image1,
-      currentImageCounter: 0,
-    };
-  }
-
-  async componentDidMount() {
-    this.interval = setInterval(() => {
-      this.setState((prevState) => ({
-        currentImageCounter:
-          (prevState.currentImageCounter + 1) % imagesPreload.length,
-        currentImage: imagesPreload[prevState.currentImageCounter],
-      }));
+const HeadArticle = () => {
+  const [currentImage, setCurrentImage] = useState(0);
+
+  useEffect(() => {
+    const imageChange = setInterval(() => {
+      setCurrentImage(current =>
+        current === imagesPreload.length - 1 ? 0 : current + 1,
+      );
     }, 5000);
```

Comment:
Why you need to use `setInterval`?

Reply:
Because image is changing like slide show